src/couchapps/WMStats/_attachments/js/DataStruct/T1/WMStats.RequestSummary.js

```diff
@@ -137,3 +137,24 @@ WMStats.Requests = function(data) {
     };
     return tier1Requests;
 };
+
+
+WMStats.Requests.getPropertyByTask = function(taskPath, property, requestInfo) {
+	
+    if (requestInfo.TaskChain) {
+        var numTasks = requestInfo.TaskChain;
+        var taskSplit = taskPath.split("/");
+        var task = taskSplit[taskSplit.length -1];
+        for (var i=1; i <= numTasks; i++) {
+            if (requestInfo["Task" + i].TaskName == task) {
+                if (requestInfo["Task" + i][property] !== undefined) {
```

Comment:
Yes, default value inside the task would be not good. That means it will overwrite the workflow level default. I think if something sets Null in task property most likely that is wrong on the config part.  But if some one said intentionally we have respect that as that property is set to Null for some reason.